### ISSUE TITLE
Cypress error with React project with javascript (--js flag)

### ISSUE THREAD

**Issue by alexpthomasdev** [nrwl/nx#2964]
_[Please make sure you have read the submission guidelines before posting an issue](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#-submitting-issue)_

# Prerequisites

Please answer the following questions for yourself before submitting an issue.
**YOU MAY DELETE THE PREREQUISITES SECTION.**

- [X ] I am running the latest version
- [X ] I checked the documentation (nx.dev) and found no answer
- [X ] I checked to make sure that this issue has not already been filed
- [X ] I'm reporting the issue to the correct repository (not related to React, Angular or any dependency)

## Expected Behavior

I'm expecting to run the cypress test on my react project without an error when I create a project using only javascript (no Typescript)

## Current Behavior

Currently, when I run my e2e project (nx run project-e2e:e2e) it returns this error
```
Oops...we found an error preparing this test file:

  src\integration\app.spec.js

The error was:

Error: Webpack Compilation Error
C:\Repos\frontend-nx-spike\apps\profile-e2e\tsconfig.e2e.json
[tsl] ERROR
      TS18003: No inputs were found in config file 'tsconfig.json'. Specified 'include' paths were '["src/**/*.ts","src/**/*.js"]' and 'exclude' paths were '["../../node_modules","../../tmp"]'.
 @ multi ./src/integration/app.spec.js main[0]

./src/integration/app.spec.js
Module build failed (from C:/Repos/frontend-nx-spike/node_modules/ts-loader/index.js):
Error: error while parsing tsconfig.json
 @ multi ./src/integration/app.spec.js main[0]

This occurred while Cypress was compiling and bundling your test code. This is usually caused by:

- A missing file or dependency
- A syntax error in the file or one of its dependencies

Fix the error in your code and re-run your tests.

Oops...we found an error preparing this test file:

  src\support\index.js

The error was:

Error: Webpack Compilation Error
C:\Repos\frontend-nx-spike\apps\profile-e2e\tsconfig.e2e.json
[tsl] ERROR
      TS18003: No inputs were found in config file 'tsconfig.json'. Specified 'include' paths were '["src/**/*.ts","src/**/*.js"]' and 'exclude' paths were '["../../node_modules","../../tmp"]'.
 @ multi ./src/support/index.js main[0]

./src/support/index.js
Module build failed (from C:/Repos/frontend-nx-spike/node_modules/ts-loader/index.js):
Error: error while parsing tsconfig.json
 @ multi ./src/support/index.js main[0]

This occurred while Cypress was compiling and bundling your test code. This is usually caused by:

- A missing file or dependency
- A syntax error in the file or one of its dependencies

Fix the error in your code and re-run your tests.
```

## Failure Information (for bugs)

I double checked the tsconfig.json and the tsconfig.e2e.json and I can see that is looking for files ending on '.js' and '.ts'

I discovered that if I rename my `app.spec.js` file to `app.spec.ts` it works, but all this files are auto generated and I expect not to change them on any project, it should be working from scratch.

### Steps to Reproduce

Create a workspace project empty.
Create a new project for React using the --js flag
This will generate the project-e2e project.

Try to run the e2e test from your console `nx run project-e2e:e2e`

I expect to work but it shows a configuration error described previously

### Context

Please provide any relevant information about your setup:

  @nrwl/angular : Not Found
  @nrwl/cli : 9.2.3
  @nrwl/cypress : 9.2.3
  @nrwl/eslint-plugin-nx : 9.2.3
  @nrwl/express : Not Found
  @nrwl/jest : 9.2.3
  @nrwl/linter : 9.2.3
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : 9.2.3
  @nrwl/schematics : Not Found
  @nrwl/tao : 9.2.3
  @nrwl/web : 9.2.3
  @nrwl/workspace : 9.2.3
  typescript : 3.8.3

**Comment by FrozenPandaz** [comment]
This is actually already fixed in `master`.

Please recheck with the `^9.3.0-beta.3` to see if this has been solved there.

**Comment by alexpthomasdev** [comment]
I updated to the version 9.3.0 and the error still there.

`nx report`
```  @nrwl/angular : Not Found
  @nrwl/cli : 9.3.0
  @nrwl/cypress : 9.3.0
  @nrwl/eslint-plugin-nx : 9.3.0
  @nrwl/express : Not Found
  @nrwl/jest : 9.3.0
  @nrwl/linter : 9.3.0
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : 9.3.0
  @nrwl/schematics : Not Found
  @nrwl/tao : 9.3.0
  @nrwl/web : 9.3.0
  @nrwl/workspace : 9.3.0
  typescript : 3.8.3
```
I try to run nx run project-e2e:e2e. My file is in project-e2e/src/integration/app.spec.js

and tsconfig.e2e.json is 

```

**Comment by FrozenPandaz** [comment]
Thank you for retrying with the latest version. Is it possible for you to provide a repo for us to help debug this?

@juristr I thought this would have been fixed.

**Comment by juristr** [comment]
I can give this a look. @FrozenPandaz not sure this is related to what we were looking at.

**Comment by alexpthomasdev** [comment]
Ok, so I downloaded the latest version (9.3.0) and created a new project from scratch and I can see that there it works.
I was migrating from the 9.2.3.
I was comparing the config files from both repos and I think I found the difference.

If somebody has the same issue, the solution is change the tsconfig.e2e.json file and add "allowJS":true on the compilerOptions

```
{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "sourceMap": false,
    "outDir": "../../dist/out-tsc",
    "allowJs": true
  },
  "include": ["src/**/*.ts", "src/**/*.js"]
}
```

So problem solve :)

Thanks for your help!
